<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>138</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>138</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100</width>
    <height>138</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>115</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>115</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 0px;
font-family: -apple-system,BlinkMacSystemFont,&quot;Malgun Gothic&quot;,&quot;맑은고딕&quot;,helvetica,&quot;Apple SD Gothic Neo&quot;,sans-serif;
font-size: 12px;</string>
     </property>
     <property name="lineWrapColumnOrWidth">
      <number>17</number>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>댓글수              조회수              좋아요수              글자수               사진수               *띄어쓰기 주의!*    *괄호 가능*</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="custom_btn">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>23</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  background-color: rgb(58, 59, 61);
  color: rgb(101,201,117);
  border-radius: 10px;
  font-family: -apple-system,BlinkMacSystemFont,&quot;Malgun Gothic&quot;,&quot;맑은고딕&quot;,helvetica,&quot;Apple SD Gothic Neo&quot;,sans-serif;
  font-size: 15px;
  font-weight: bold;
}

QPushButton:hover {
  background-color: rgb(89, 91, 94);
}

QPushButton:pressed {
  background-color: rgb(58, 59, 61);
}</string>
     </property>
     <property name="text">
      <string>수식 검색</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
